src/skins/vector/css/matrix-react-sdk/views/rooms/_AudioRecorder.scss

```diff
@@ -0,0 +1,24 @@
+.mx_MessageComposer_ptt {
+  padding-right: 5px;
+  cursor: pointer;
+  padding-top: 4px;
+
+  object {
```

Comment:
This looks like it's for preventing clicks to a flash object in the PTT button, although there won't be one now we're using MediaRecorder.